client/src/components/BuildingStatsTable.tsx

```diff
@@ -10,250 +10,359 @@ import JFCLLinkInternal from "./JFCLLinkInternal";
 import { Link } from "react-router-dom";
 import "styles/BuildingStatsTable.css";
 import { removeIndicatorSuffix } from "routes";
+import { Icon } from "@justfixnyc/component-library";
+import Modal from "./Modal";
 
 interface BuildingStatsAddrContext {
   getBuildingStats(): AddressRecord;
 }
 
-const AddrContext = React.createContext<BuildingStatsAddrContext>({
-  getBuildingStats: () => {
-    throw new Error("This Building Stats table has no data!");
-  },
-});
+const BuildingStatsTableWithoutI18n = (props: { addr: AddressRecord; timelineUrl: string }) => {
+  const [showInfoModal, setShowInfoModal] = React.useState(false);
+  const [infoModalTitle, setInfoModalTitle] = React.useState("");
+  const [infoModalContent, setInfoModalContent] = React.useState("");
 
-const bblDash = (
-  <span className="unselectable" unselectable="on">
-    -
-  </span>
-);
+  const AddrContext = React.createContext<BuildingStatsAddrContext>({
+    getBuildingStats: () => {
+      throw new Error("This Building Stats table has no data!");
+    },
+  });
 
-const delta = "Δ";
-
-const BBL = () => {
-  const addr = useContext(AddrContext).getBuildingStats();
-  const { boro, block, lot } = Helpers.splitBBL(addr.bbl);
-  return (
-    <I18n>
-      {({ i18n }) => (
-        <div
-          className="double"
-          title={i18n._(
-            t`This is the official identifer for the building according to the Dept. of Finance tax records.`
-          )}
-        >
-          <label>
-            Boro{bblDash}Block{bblDash}Lot (BBL)
-          </label>
-          {boro}
-          {bblDash}
-          {block}
-          {bblDash}
-          {lot}
-        </div>
-      )}
-    </I18n>
+  const bblDash = (
+    <span className="unselectable" unselectable="on">
+      -
+    </span>
   );
-};
 
-const YearBuilt = () => {
-  const addr = useContext(AddrContext).getBuildingStats();
-  return (
-    <I18n>
-      {({ i18n }) => (
-        <div
-          title={i18n._(
-            t`The year that this building was originally constructed, according to the Dept. of City Planning.`
-          )}
-        >
-          <Trans render="label">Year Built</Trans>
-          {addr.yearbuilt !== 0 ? addr.yearbuilt : "N/A"}
-        </div>
-      )}
-    </I18n>
-  );
-};
+  const BBL = () => {
+    const addr = useContext(AddrContext).getBuildingStats();
+    const { boro, block, lot } = Helpers.splitBBL(addr.bbl);
+    const description =
+      "This is the official identifer for the building according to the Dept. of Finance tax records";
```

Comment:
Do these descriptions for aria-labels need to use lingui for translations as well?

Reply:
oh wow that's all great to know, thanks for looking into that! That makes sense to make those smaller changes now and work on reviewing everything else later.